### PR TITLE
History and intro typos and improvements

### PR DESCRIPTION
**Pull request by pavpanchekha**
Not all of these are perhaps "worth" making or maybe @chrishtr you won't agree with them, but I wanted to raise all possible options for improving the text, and then we could filter out later.

### CODE REVIEW

**Comment by pavpanchekha** [comment]
Apologies for two chapters in one PR, I messed up with git.

File: book/history.md [hunk 1]
```diff
@@ -326,10 +326,11 @@ on, the HTML specification ended up moving to a different standards body called
...
 the W3C. JavaScript\index{JavaScript} is
-standardized at [TC39](https://tc39.es/)\index{TC39}
-(“Technical Committee 39” at
-[ECMA](https://www.ecma-international.org/about-ecma/history/), yet another
-standards body). [HTTP](https://tools.ietf.org/html/rfc2616) is standardized by
+standardized at yet another standards body,
+[TC39](https://tc39.es/)\index{TC39}
+(“Technical Committee 39”) at
+[ECMA](https://www.ecma-international.org/about-ecma/history/).
+[HTTP](https://tools.ietf.org/html/rfc2616) is standardized by
```
Comment (pavpanchekha) [inline book/history.md:333]: This changes
> standardized at TC39 [XX] ("Technical Committee 39" at ECMA, yet another standard body.
to
> standardized at yet another standards body, TC39 ("Technical Committee 39" [XX]) at ECMA.
Sounds better to my ear, avoid ugly typography with parentheses / citations.

File: book/history.md [hunk 2]
```diff
@@ -359,7 +360,7 @@ Object Model you'll learn about in this book. Netscape and Internet Explorer at
 first had incompatible implementations of these features, and it took years,
 the development of a common specification, and significant pressure campaigns on
 the browsers before standardization was achieved. You can read about this story in
-much more depth [here](https://css-tricks.com/chapter-7-standards/).
+much more depth [from Jay Hoffman](https://css-tricks.com/chapter-7-standards/).
```
Comment (pavpanchekha) [inline book/history.md:363]: This looks bad in the printed book because "here" refers to nothing.

File: book/http.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ next: graphics
 ...
 
 A web browser displays information identified by a URL. And the first
-step is to use that URL to connect to and download that information
-from a server somewhere on the Internet.
+step is to use that URL to connect to and download information from a
+server somewhere on the Internet.
```
Comment (pavpanchekha) [inline book/http.md:10]: This sounds better to my ear.

File: book/http.md [hunk 4]
```diff
@@ -141,7 +141,7 @@ Requesting Information
 Once it's connected, the browser requests information from the server
 by giving its *path*, the path being the part of a URL that comes
 after the host name, like `/index.html`. The structure of the request
-is shown in Figure 2. You can type this into `telnet` to try it.
+is shown in Figure 2. Type this into `telnet` to try it.
```
Comment (pavpanchekha) [inline book/http.md:144]: Later on, we use the imperative (line 182 in the source, "So type a blank line into `telnet`") so I think we should use it here too.

File: book/http.md [hunk 5]
```diff
@@ -234,8 +234,7 @@ This tells you that the host confirms that it, too, speaks `HTTP/1.0`,
 and that it found your request to be "OK" (which has a numeric code of
 200). You may be familiar with `404 Not Found`; that's another numeric
 code and response, as are `403 Forbidden` or `500 Server Error`. There
-are lots of these codes,^[As [this flow
-chart](https://github.com/for-GET/http-decision-diagram) shows.]
+are lots of these codes,
```
Comment (pavpanchekha) [inline book/http.md:237]: I don't think this footnote is helpful. In a later "Go Further" we refer to Wikipedia page for a list of all codes.

File: book/http.md [hunk 6]
```diff
@@ -553,16 +552,16 @@ endings.
     `Content-Type` header usually contains a `charset` declaration
     that specifies the encoding of the body. If it's absent, browsers
     still won't default to `utf8`; they'll guess, based on letter
-    frequencies, and you see ugly � strange áççêñ£ß when they guess
-    wrong. Incorrect-but-common `utf8` skips all that complexity.
```
Comment (pavpanchekha) [inline book/http.md:557]: I don't know what this last sentence means

File: book/http.md [hunk 7]
```diff
@@ -553,16 +552,16 @@ endings.
...
+    frequencies, and you will see ugly � strange áççêñ£ß when they guess
```
Comment (pavpanchekha) [inline book/http.md:555]: Otherwise the tense I think is not consistent.

File: book/http.md [hunk 8]
```diff
@@ -659,8 +658,8 @@ servers seem to use together with compression.
 Displaying the HTML
 ===================
 
-The HTML code in the body defines the content you see in your browser
-window when you go to <http://example.org/index.html>. I'll be
+The HTML code in the response body defines the content you see in your browser
+window when you go to `<http://example.org/index.html>`. I'll be
```
Comment (pavpanchekha) [inline book/http.md:662]: Here the link should, I think, be distinguished somehow. Typewriter font is fine, as is an underline. I'm not sure if this actually renders right.

File: book/http.md [hunk 9]
```diff
@@ -727,6 +726,7 @@ if __name__ == "__main__":
 The first line is Python's version of a `main` function, run only when
 executing this script from the command line. The code reads the first
 argument (`sys.argv[1]`) from the command line and uses it as a URL.
+
 Try running this code on the URL `http://example.org/`:
```
Comment (pavpanchekha) [inline book/http.md:730]: I think this makes more sense as a separate paragraph.

File: book/http.md [hunk 10]
```diff
@@ -915,7 +915,7 @@ This chapter went from an empty file to a rudimentary web browser that
 can:
 
 -   parse a URL into a scheme, host, port, and path;
--   connect to that host using the `sockets` and `ssl` libraries;
+-   connect to that host using the `socket` and `ssl` libraries;
```
Comment (pavpanchekha) [inline book/http.md:918]: Major mistake!

**Review by pavpanchekha — COMMENTED** [review]
Explanations

File: book/http.md (hunk 3 above)
Comment (chrishtr) [inline book/http.md:10]:
```suggestion
server somewhere on the internet.
```

(used this way elsewhere)

File: book/http.md (hunk 6 above)
Comment (chrishtr) [reply book/http.md:557]: I think it means "`utf8` is common enough that our browser assuming it works well enough for our purposes." Let's leave this sentence in.

File: book/http.md (hunk 8 above)
Comment (chrishtr) [inline book/http.md:662]:
```suggestion
window when you go to `http://example.org/index.html`. I'll be
```

File: book/http.md (hunk 9 above)
Comment (chrishtr) [reply book/http.md:730]: Seems not critical to fix.

**Review by chrishtr — APPROVED** [review]

**Pull request merged**